source/datasets/weather.rst

```diff
@@ -8,5 +8,5 @@ Output          JSON, mime-type application/json
 E-mail contact  |contact|
 API version     :strike:`v1` |deprecated|, v2
 Swagger URL     |apiv2|
-|station|       MeteoStation
+:literal:`StationType`     MeteoStation
```

Comment:
@stefanodavid Why you do this now? Why not just use `|stationtype|`, what if we change that parameter in the future?

Reply:
Peter I am sorry but I am not understand what the hell you want me to do. Let me recall:

You say that using `|station|` as a replacement for `StationType` or, better 
```
:literal:`StationType`
```
may be ambiguous for a reader and suggested to use `|stationtype|` so it would be clearer.

I disagreed with you and brought my explanation, but you wanted me to do the change anyway from 
```
.. |station| replace:: :literal:`StationType`
```
```
.. |stationtype| replace:: :literal:`StationType`
```
to avoid any ambiguity for any kind of readers/users/whoever.

To me such a change is needless and useless, and I'm gonna explain why in a while, but I incorporated nonetheless the changes you required and now the source code is unambiguous. Well, unless you tell me that using the `:literal:` role to emphasise the word is ambiguous... :-)

However, now that I changed the source code according to you requirements, you wonder what the hell will I do if or when you change the parameter name. The answer is simple and explains why :
In order to replace `|station|` throughout the code with `|stationtype|` I need not only to replace statement as described above, but also to recursively: 
```
sed -i 's/|station|/|stationtype|`
```
or
```
sed -i 's/|station|/:literal:`StationType`/' *.rst
```
which is what I did, but then I had also got to **manually edit files** to trim spaces, because they occur within tables and indentation would be screwed. At lest, I do not now how to trim spaces within one line automatically.

In case you should decide to rename for example StationType to StationModel (please replace StationModel with whatever string you want), I sincerely **fail to understand** why 
```
.. |stationtype| replace:: :literal:`StationModel`
```
would in that case not cause any ambiguities...  prefer. Wouldn't you require an additional change? 

At this point, I think I prefer to write the documentation for the Open Data Hub Project directly in HTML, so there is no includes, no replacements, no other means to waste time in endless and useless discussions like this one, everybody including reviewers can access docs the same way, there's no need to build documentation from source code and so on. Please decline this PR, then provide guidelines on how should I proceed, thank you. I would not be following this discussion any further, it already cost me too much time to try to  meet your requirements.